packages/suite/package.json

```diff
@@ -10,7 +10,7 @@
         "translations:format": "yarn prettier --write \"../suite-data/files/translations/*\"",
         "translations:extract": "formatjs extract src/support/messages.ts --format simple > ../suite-data/files/translations/master.json",
         "translations:upload": "crowdin upload",
-        "translations:download": "crowdin download --all && translations:format",
```

Comment:
I am not sure about removing format, if anyone will call download manually then will be confused why messages are not formatted correctly

Reply:
Maybe there is a better way but I don't see a problem with running it separately if you'd like to do it locally so `yarn workspace @trezor/suite translations:download --token=<token> && yarn workspace @trezor/suite translations:format`

Reply:
If we are not going to run this locally then this is probably not big issue